index.html

```diff
@@ -1,163 +1,273 @@
+            </li>
+          </ul>
+          <a><button type="button">Get my resume</button></a>
+
+          <div class="languages">
+            <h3>Languages</h3>
+            <i class="fa-solid fa-chevron-down fa-xl"></i>
+          </div>
+          <div class="language">
+            <img
+              class="language-img"
+              src="./img/js.png"
+              alt="JavaScript image"
+            />
+            <p>JavaScript</p>
+          </div>
+          <div class="language">
+            <img
+              class="language-img"
+              src="./img/html.jpg"
+              alt="JavaScript image"
+            />
+            <p>HTML</p>
+          </div>
+          <div class="language">
+            <img
+              class="language-img"
+              src="./img/css.jpg"
+              alt="JavaScript image"
+            />
+            <p>CSS</p>
+          </div>
+          <div class="languages">
+            <h3>Frameworks</h3>
+            <i class="fa-solid fa-chevron-right fa-xl"></i>
+          </div>
+          <div class="languages">
+            <h3>Skills</h3>
+            <i class="fa-solid fa-chevron-right fa-xl"></i>
+          </div>
         </div>
-        <div class="card">
-            <img class="card-image" src="./img/portfolio2.jpg" alt="webpage image">
-            <h2>Tonic</h2>
-            <div class="tags">
-                <p>CANOPY</p>
-                <img class="bullet" src="./img/Counter.png" alt="bullet">
-                <span>Back End Dev</span>
-                <img class="bullet" src="./img/Counter.png" alt="bullet">
-                <span>2015</span>
-            </div>
-            <p class="description">
-                A daily selection of privately personalized reads. No accounts or 
-                sign-ups required.
+      </div>
+      </section>
+      <section class="contact">
+        <div class="contact-div" >
+          <div class="div-content">
+          <div class="contact-h1">
+            <h1>Contact me</h1>
+          </div>
+          <div class="contact-text">
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I'd
+              love to help with it.
             </p>
-            <ul class="technologies">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-            </ul>
+          </div>
+
+          <form
+            class="form"
+            action="https://formspree.io/f/xeqwolgk"
+            method="post"
+          >
             <div>
-                <button type="button">See Project</button>
+              <fieldset>
+                <input
+                  type="text"
+                  placeholder="Yeremias"
+                  maxlength="30"
+                  required
+                />
+              </fieldset>
             </div>
```

Comment:
[OPTIONAL] Good job trying to lay out the input fields within a wrapper but from a different perspective it is not necessary to do this because you are already using flex to align all of the inputs, so I am suggesting that you remove all the wrapper `div` to keep your code well structured and compact.